docs/evaluation/tutorials/eval_testing_tutorial.mdx

```diff
@@ -0,0 +1,813 @@
+    },
+    {
+      value: "vitest",
+      label: "Vitest",
+      language: "typescript",
+      content: `// THIS CODE GOES OUTSIDE THE TEST - IT IS JUST A HELPER FUNCTION
+const judgeLLM = new ChatOpenAI({ model: "gpt-4o" });
+
+const groundedEvaluator = async (params: {
+    answer: string;
+    referenceDocuments: string, 
+  }) => {
+    const instructions = [
+      "Return 1 if the ANSWER is grounded in the DOCUMENTS",
+      "Return 0 if the ANSWER is not grounded in the DOCUMENTS",
+    ].join("\\n");
+    const grade = await judgeLLM.invoke([
+      { role: "system", content: instructions },
+      { role: "user", content: \`ANSWER: \${params.answer}\\nDOCUMENTS: \${params.referenceDocuments}\` },
+    ]);
+
+    const score = parseInt(grade.content.toString());
+    return { key: "groundedness", score };
+  };
+  
+  // THIS CODE GOES INSIDE THE TEST
+  ls.test(
+    "grounded in the source",
+    {
+      inputs: { query: "How did Nvidia stock do in 2024 according to analysts?" },
+    },
+    async ({ inputs: { query } }) => {
+      const result = await agent.invoke({
+        messages: [{ role: "user", content: query }],
+      });
+
+      const wrappedEvaluator = ls.wrapEvaluator(groundedEvaluator);
+
+      await wrappedEvaluator({
+        answer: result.structuredResponse.text_answer ?? "",
+        referenceDocuments: result.structuredResponse.reasoning,
+      })
+
+      ls.logOutputs(result);
+    });`,
+    },
+    {
+      value: "jest",
+      label: "Jest",
+      language: "typescript",
+      content: `// THIS CODE GOES OUTSIDE THE TEST - IT IS JUST A HELPER FUNCTION
+const judgeLLM = new ChatOpenAI({ model: "gpt-4o" });
+
+const groundedEvaluator = async (params: {
+    answer: string;
+    referenceDocuments: string,
+  }) => {
+    const instructions = [
+      "Return 1 if the ANSWER is grounded in the DOCUMENTS",
+      "Return 0 if the ANSWER is not grounded in the DOCUMENTS",
+    ].join("\\n");
+    const grade = await judgeLLM.invoke([
+      { role: "system", content: instructions },
+      { role: "user", content: \`ANSWER: \${params.answer}\\nDOCUMENTS: \${params.referenceDocuments}\` },
+    ]);
+
+    const score = parseInt(grade.content.toString());
+    return { key: "groundedness", score };
+  };
+  
+  // THIS CODE GOES INSIDE THE TEST
+  ls.test(
+    "grounded in the source",
+    {
+      inputs: { query: "How did Nvidia stock do in 2024 according to analysts?" },
+    },
+    async ({ inputs: { query } }) => {
+      const result = await agent.invoke({
+        messages: [{ role: "user", content: query }],
+      });
+
+      const wrappedEvaluator = ls.wrapEvaluator(groundedEvaluator);
+
+      await wrappedEvaluator({
+        answer: result.structuredResponse.text_answer ?? "",
+        referenceDocuments: result.structuredResponse.reasoning,
+      })
+
+      ls.logOutputs(result);
+    });`,
+    },
+  ]}
+  groupId="client-language"
+/>
+
+## Run tests
+
+### Config file for Vitest
```

Comment:
I've updated these in the how-to here:

https://langsmith-docs-git-bagatur-pytestdocs-langchain.vercel.app/evaluation/how_to_guides/vitest_jest#vitest